include/osdetection

```diff
@@ -180,6 +180,13 @@
                             OS_VERSION_FULL=$(grep "^VERSION=" /etc/os-release | awk -F= '{print $2}' | tr -d '"')
                             OS_NAME="Debian"
                         ;;
+                        "pureos")
+                            LINUX_VERSION="PureOS"
+                            OS_FULLNAME=$(grep "^PRETTY_NAME=" /etc/os-release | awk -F= '{print $2}' | tr -d '"')
```

Comment:
instead of `grep | awk | tr`, how about awk `awk -F= '/^PRETTY_NAME=/ {print substr($2,2,length($2)-2)}' /etc/os-release`?

Reply:
Thanks Nick, that is a good suggestion. Will apply that as a default to avoid repetition.